### PR TITLE
Resolved bootstrap conflict, added fashrevwall project.

### PR DESCRIPTION
**Pull request by lili2311**
@womenhackfornonprofits/whfnp
## What are the changes included in this PR?
- added Fashion Revolution to Projects
- fixed bootstrap path conflict booboo
  ## If the changes are visual, provide a screenshot.
  screen shot 2016-05-13 at 23 52 39

### CODE REVIEW

File: src/projects.html [hunk 1]
```diff
@@ -1,93 +1,7 @@
...
-              <a href="https://blog.womenhackfornonprofits.com/">Blog</a>
-            </li>
-            <li>
-              <a href="contact.html">Contact</a>
-            </li>
-          </ul>
-        </nav>
-      </div>
-    </header>
+---
```
Comment (tanyapowell) [inline src/projects.html:1]: Have you also added templating to this page?

File: src/sass/partials/_bootstrap-custom.scss [hunk 2]
```diff
@@ -5,40 +5,24 @@
...
 // Reset and dependencies
 @import "bootstrap/normalize";
-=======
-@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables";
-@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins";
 
-// Reset and dependencies
-@import "../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/normalize";
->>>>>>> 4d2d4413c73830ad76c174dffda6239fa646d6c1:src/sass/partials/_bootstrap.scss
 // @import "bootstrap/print";
```
Comment (tanyapowell) [inline src/sass/partials/_bootstrap-custom.scss]: Should these commented out imports be deleted? Or are they to be used at a later date?

Comment (lili2311) [inline src/sass/partials/_bootstrap-custom.scss]: sorry this was the git merge conflict residue that got missed, the actual file is now correct. the ones pushed in the PR

File: src/projects.html (hunk 1 above)
Comment (lili2311) [inline src/projects.html:1]: I can add it in a PR after yours is merged so I can re-use what you extracted?

File: src/sass/partials/_bootstrap-custom.scss (hunk 2 above)
Comment (tanyapowell) [inline src/sass/partials/_bootstrap-custom.scss]: I meant all the lines commented out such as line 16 `// @import "bootstrap/print";`. If we aren't using them should just delete them

File: src/projects.html (hunk 1 above)
Comment (tanyapowell) [inline src/projects.html:1]: You just added the default layout?

**Comment by EChesters** [comment]
We should avoid putting images in the CSS, because the images are content that needs to be added via HTML. I think there is an issue to replace all images (other than banner) like this already. 

Title in projects.html shouldn't have quote marks around it.

Could you also remove the notes from the commit title?

**Comment by lili2311** [comment]
Yep there is a separate issue open for that, so can change all at once on the next PR.

Will fix the other comments and push an update.

**Comment by lili2311** [comment]
@tanyapowell  and @EChesters  can I merge this and do the templating in the next PR this PR is older than the templates.

**Comment by tanyapowell** [comment]
@lili2311 could you have a quick look at the conflicts first? You won't be able to merge with them in

**Comment by lili2311** [comment]
done!

**Comment by lili2311** [comment]
please OK :) @tanyapowell or @EChesters  and I will merge

**Comment by tanyapowell** [comment]
👍

**Pull request merged**